simpler_eda/numerical_eda.py

```diff
@@ -80,5 +74,146 @@ def numerical_plot(
                  title = " Horsepower vs Acceleration",
                  font_size = 10)
     """
-
-    return numerical_plot
+    # Defensive programming: Check if user provides valid inputs
+
+    # Ensure title variable is specified by the user
+    assert title is not None, "Variable title needs to be specified."
```

Comment:
if the title is a must, shall we change it to the required argument? Or We update the none to a default title base on a base on the x and y value they input (e.g. Horsepower vs Acceleration Line Plot)

Reply:
I like updating it to default for it. Will work on that